common/minimodel/site.py

```diff
@@ -35,5 +35,7 @@ def __init__(self, site_name: str, coordinate_center: str, astropy_lookup: Optio
             msg = f'Unknown time zone lookup: {timezone_info}.'
             raise ValueError(e, msg)
 
+    def __repr__(self) -> str:
+        return f"SiteInformation('{self.name}', '{self.coordinate_center}')"
```

Comment:
We should get rid of this now since `SiteInformation` is gone and it doesn't make sense any more.

Reply:
Got mixed up in the rebase